remote-access/ssh/passwordless.md

```diff
@@ -10,7 +10,7 @@ First, check whether there are already keys on the computer you are using to con
 ls ~/.ssh
 ```
 
-If you see files named `id_rsa.pub` or `id_dsa.pub` you have keys set up already, so you can skip the generating keys step (or delete these files with `rm id*` and make new keys).
+If you see files named `id_rsa.pub` or `id_dsa.pub` then you have keys set up already, so you can skip the 'Generate new SSH keys' step (or delete these files with `rm id*` and make new keys).
```

Comment:
Why is this suggesting deleting the old keys? Is it worth adding a warning here that deleting the old keys _might_ lock the user out of other systems if those keys are being used elsewhere?
(Or is this one of those situations where if the keys _were_ already being used, the user would know they shouldn't delete them?)

Reply:
I've no idea. Seems a bit daft to me. I will remove that bit as it is both unnecessary, and liable to trip up noobs.